src/sp_config.c

```diff
@@ -61,6 +62,26 @@ int parse_empty(char *restrict line, char *restrict keyword, void *retval) {
   return 0;
 }
 
+int parse_list(char *restrict line, char *restrict keyword, void *list_ptr) {
```

Comment:
What about moving this function into `sp_list.c` instead?

Reply:
It is used just like `parse_str`, `parse_empty` in the parser, so I think it is better in this file.